.github/workflows/test-pr.yml

```diff
@@ -98,6 +98,7 @@ jobs:
           repository: kframework/k-exercises
           token: ${{ secrets.JENKINS_GITHUB_PAT }}
           submodules: recursive
+          ref: 'imp++'
```

Comment:
Why is this an issue? And how is it helping?

Reply:
this is just for testing purposes because we need to test against the modified version of the exercises. I will revert this change before merging.